Wheel integration

citlaligm opened the pull request:
This PR creates a page for readthedocs as a tutorial of how to integrate a wheel on the LV client.

Read the docs build:
https://monodrive.readthedocs.io/en/wheel_integration/LV_client/tutorials/wheel_integration/

closes #117

citlaligm commented:
This PR creates a page for readthedocs as a tutorial of how to integrate a wheel on the LV client.

Read the docs build:
https://monodrive.readthedocs.io/en/wheel_integration/LV_client/tutorials/wheel_integration/

katiemacaulay commented on docs/LV_client/tutorials/wheel_integration.md:
```diff
@@ -0,0 +1,31 @@
+# Wheel integration
+
```
Can we add something here about Wheel Integration, something like: Labview's platform enables users to plug-in(there's probably a better word for that) a wheel (specific wheel or any wheel?) in the simulation. In the vi package manager, there two examples are available for use with a wheel: mono_closed_loop_example_wheel.vi and mono_replay_example_wheel.vi. (add more details here about hardware options if you'd like)

katiemacaulay commented on docs/LV_client/tutorials/wheel_integration.md:
```diff
@@ -0,0 +1,31 @@
+# Wheel integration
```
Capitalize Integration

katiemacaulay replied:
Speaking of, would it be possible to go ahead and capitalize all the tab titles like this?

katiemacaulay commented on docs/LV_client/tutorials/wheel_integration.md:
```diff
@@ -0,0 +1,31 @@
+# Wheel integration
+
+## Pre-requisites
+1. The **mono_closed_loop_example_wheel.vi** and the **mono_replay_example_wheel.vi** use the **G920 for Xbox One and PC**, more information [here](https://www.logitechg.com/en-us/products/driving/driving-force-racing-wheel.html).
```
Are there any other .vi programs the user can use or create on their own? If not, making this explicit in the introduction could really clean up this section such as: 1.  **G920 for Xbox One and PC**, more information [here](https://www.logitechg.com/en-us/products/driving/driving-force-racing-wheel.html).

katiemacaulay commented on docs/LV_client/tutorials/wheel_integration.md:
```diff
@@ -0,0 +1,31 @@
+# Wheel integration
+
+## Pre-requisites
```
Just for consistency with the getting started page : no dash: Prerequisites

katiemacaulay commented on docs/LV_client/tutorials/wheel_integration.md:
```diff
@@ -0,0 +1,31 @@
+
+## Pre-requisites
+1. The **mono_closed_loop_example_wheel.vi** and the **mono_replay_example_wheel.vi** use the **G920 for Xbox One and PC**, more information [here](https://www.logitechg.com/en-us/products/driving/driving-force-racing-wheel.html).
+1. Install the Logitech G HUB to obtain the latest drivers for the wheel from [here](https://www.logitechg.com/en-us/innovation/g-hub.html)
```
Maybe try: Ensure you have the latest drivers for the wheel from [Logitech G HUB](https://www.logitechg.com/en-us/innovation/g-hub.html)

katiemacaulay commented on docs/LV_client/tutorials/wheel_integration.md:
```diff
@@ -0,0 +1,31 @@
+1. Install the Logitech G HUB to obtain the latest drivers for the wheel from [here](https://www.logitechg.com/en-us/innovation/g-hub.html)
+1. Download the Logitech Steering Wheel SDK from [here](https://www.logitechg.com/en-us/innovation/developer-lab.html)
+1. Create a LabVIEW library using the **LogitechSteeringWheelEnginesWrapper.dll** using the Import Shared Library Wizard, you can find a tutorial on how to do this [here](http://zone.ni.com/reference/en-XX/help/371361R-01/lvhowto/example_import_shared_library/).
+1. Install the **Maker Hub Interface for Xbox One Controller** using the VIPM.
```
Just for extra clarification: VI Package Manager.

katiemacaulay commented on docs/LV_client/tutorials/wheel_integration.md:
```diff
@@ -0,0 +1,31 @@
+## Closed Loop with Wheel 
+<div class="img_container">
+  <video width=650px height=440px muted controls autoplay loop>
+    <source src="http://cdn.monodrive.io/readthedocs/closed-loop.mp4" type="video/mp4">
```
This is amazing. The monitors are a little off because one is higher in the scene than the other. But otherwise, looks great!!! This is such a neat tool.

citlaligm commented:
@katiemacaulay thanks for the review. I implemented your suggestions. I think is ready for another look.

citlaligm commented:
@katiemacaulay thanks for the review. I implemented your suggestions. I think is ready for another look.

I also update the video links in the tutorial section and I added the links missing.

katiemacaulay commented on docs/LV_client/tutorials/AddSensors.md:
```diff
@@ -1,3 +1,6 @@
-## Add new sensors
+## Add New Sensors
 Learn to add a camera sensor using the labVIEW Client.  The same procedure can be followed to add multiple Radar, Lidar, and Ultransonic sensors.  Sensors are placed with respect to the vehicle's origin.
```
followed to add multiple Radar, **LiDAR**, and **Ultrasonic** sensors.

katiemacaulay replied:
Also- LabVIEW should be capitalized.

katiemacaulay commented on docs/LV_client/tutorials/Simulator_configuration.md:
```diff
@@ -0,0 +1,9 @@
+## Simulator Configuration
+
+This tutorial demonstrates how to use the LabVIEW client to configure the behavior of vehicles in the scene as well as the object's materials when using the monoDrive Simulator. 
```
This tutorial demonstrates how to configure the behavior of vehicles and an object's materials when using the monoDrive Simulator.

katiemacaulay commented on docs/LV_client/tutorials/Simulator_configuration.md:
```diff
@@ -0,0 +1,9 @@
+## Simulator Configuration
+
+This tutorial demonstrates how to use the LabVIEW client to configure the behavior of vehicles in the scene as well as the object's materials when using the monoDrive Simulator. 
+C++, Python, ROS, Simulink, and Matlab client are also available.  
```
I think it's alright to eliminate the "also here: C++, Python, ROS, Simulink, and Matlab client are available.

katiemacaulay commented on docs/LV_client/tutorials/Weather.md:
```diff
@@ -0,0 +1,7 @@
+## Change Weather
+
+Learn how to change the weather using the LabVIEW Client. The monoDrive simulator allows you to simulate clear days, sunny days, sunsets, rain, and fog. 
```
Learn how to change the weather using the LabVIEW Client. The monoDrive simulator allows you to simulate the any of the following weather conditions during the day or at sunset.: sunny, cloudy, rain, fog, etc.

katiemacaulay commented on docs/LV_client/tutorials/Setup.md:
```diff
@@ -2,5 +2,5 @@
 This tutorial demonstrates the process of installing and running monoDrive in a LabVIEW environment. 
```
Capitalize Client in the title Setup a LabVIEW Client
